editors/vim/syntax/fluent.vim

```diff
@@ -0,0 +1,26 @@
+if exists("b:current_syntax")
+    finish
+endif
+
+syntax match fluentComment "\v#.*$"
+syntax match fluentIdentifier "^\v[a-zA-Z-]+" nextgroup=fluentDelimiter
+syntax match fluentDelimiter "\s*=\s*" contained skipnl nextgroup=fluentPattern
+syntax region fluentPattern contained start="" end="\v^(( )@!|( +[\.\[\*\}])@=)" contains=fluentPlaceableExpression
+syntax match fluentAttribute "\v\.[a-zA-Z-]+" nextgroup=fluentDelimiter
+syntax region fluentPlaceableExpression contained matchgroup=fluentPlaceableBraces start=+{+ end=+}+ contains=@fluentExpression
+
+syntax cluster fluentExpression contains=fluentBuiltin,fluentVariantEntry,fluentIdentifierExpression
+
+syntax match fluentBuiltin contained "\v[A-Z]+(\()@="
+syntax match fluentVariantEntry contained "\v\*?\[[a-z]+\]" nextgroup=fluentPattern
+syntax match fluentIdentifierExpression contained "\v[a-zA-Z-]+"
```

Comment:
Is this a `MessageReference`? Can you rename it if so, and also add `TermReference` and `VariableReference`? I think these are the most common ones.